docs/_articles/en/user-guide/byotemplate.md

```diff
@@ -4,79 +4,42 @@ lang: en
+- Add default copyright information to new files in your project.
+- Add default code to Aura or LWC to enable a new Apex class, and so on.
 
-Templates are essentially folders with files that contain your custom code. This [git repo](https://github.com/forcedotcom/salesforcedx-templates/tree/main/src/templates) contains a collection of official Salesforce templates for metadata components. Simply clone this repo, keeping the same folder structure, then update relevant template files with your code. Remove the files that you don’t wish to override.
-
-**Note:** Only updates made to the files listed here show up in source files in VS Code. There’s no such restriction when you use the CLI to specify templates for metadata objects.
-
-| Template Folder        | Default Template Files           |
-| ------------- |:-------------:|
-| apexclass     | DefaultApexClass.cls <br>_class.cls-meta.xml
-|apextrigger    | ApexTrigger.trigger <br> _trigger.trigger-meta.xml
-|Lightningapp   |DefaultLightningApp.app <br> DefaultLightningController.js <br> DefaultLightningCss.css <br> DefaultLightningHelper.js <br> DefaultLightningRenderer.js <br> DefaultLightningSVG.svg <br> DefaultLightningAuradoc.auradoc <br>auradefinitionbundle.app-meta.xml
-|Lightningcomponent |aura/default/default.cmp <br> aura/default/default.cmp-meta.xml <br> aura/default/defaultController.js <br> aura/default/defaultHelper.js <br> aura/default/defaultRenderer.js <br> aura/default/default.svg <br> aura/default/default.design <br> aura/default/default.css <br> aura/default/default.auradoc <br> lwc/default/default.js<br> lwc/default/default.html <br> lwc/default/default.js-meta.xml |
-|Lightningevent | defaultLightningEvt.evt <br> auradefinitionbundle.evt-meta.xml
-|Lightninginterface | DefaultLightningIntf.intf <br> auradefinitionbundle.intf-meta.xml
-|Lightningtest | DefaultLightningTest.resource
-|Project | Any file in the project folder
-|Staticresource | empty.resource <br> empty.js <br> empty.json <br> empty.css <br> empty.txt <br> _staticresource.resource-meta.xml
-|visualforcecomponent| DefaultVFComponent.component <br> _component.component-meta.xml
-|visualforcepage | DefaultVFPage.page <br> _page.page-meta.xml
-
-## Clone the Repo
-See GitHub documentation for instructions on how to clone repos.
+**Note**:
+This [git repo subdirectory](https://github.com/forcedotcom/salesforcedx-templates/tree/main/src/templates) contains a collection of official Salesforce templates for metadata components. Only updates made to the files listed in this directory show up in source files in VS Code. You can clone this subdirectory, or replicate it locally. Keeping the same folder structure, just update relevant template files with your code. Remove the files that you don’t wish to override.
```

Comment:
I would say it even more powerfully. You must keep the same folder structure. You can then just update relevant...

Reply:
Thanks again for the feedback. Done.